src/main/java/betterquesting/api/enums/EnumFrameType.java

```diff
@@ -0,0 +1,40 @@
+package betterquesting.api.enums;
+
+public enum EnumFrameType {
+    SQUARE,
+    ROUNDED_SQUARE, // Used as the default quest icon
+    CIRCLE,
+    HEXAGON,
+    HEXAGON90,
+    PENTAGON,
+    DIAMOND,
+    STAR4,
+    CROSS,
+    DIAGONAL_CROSS,
+    DIAGONAL_STAR4,
+    PRICKLE,
+    GEAR,
+    GATE,
+    JAGGED,
+    EGG,
+    HEART,
+    COLLAPSED_SQUARE,
+    SKULL,
+    DIAMOND_ON_SQUARE,
+    CROSS_ON_SQUARE,
+    STAR4_ON_SQUARE,
+    DIAMOND_ON_CIRCLE,
+    DIAGONAL_STAR4_ON_CROSS,
+    STAR4_ON_HEXAGON,
+    DIAGONAL_STAR4_ON_HEXAGON,
+    STAR4_ON_HEXAGON90,
+    DIAGONAL_STAR4_ON_HEXAGON90,
+    DIAGONAL_CROSS_ON_DIAMOND,
+    DIAGONAL_STAR4_ON_DIAMOND,
+    DIAMOND_ON_CROSS,
+    STAR4_ON_CROSS,
+    CROSS_ON_DIAGONAL_STAR4,
+    DIAMOND_ON_DIAGONAL_STAR4,
+    SATAR4_ON_DIAGONAL_STAR4,
```

Comment:
`SATAR4_ON_DIAGONAL_STAR4` => `STAR4_ON_DIAGONAL_STAR4`

Reply:
possibly the most timely single-letter typo fix ever? at the very least its a WR competitor, and definitely a PB.